pilot/pkg/networking/core/v1alpha3/envoyfilter/listener_patch_test.go

```diff
@@ -1324,6 +1324,7 @@ func TestApplyListenerPatches(t *testing.T) {
 									MergeSlashes:                 true,
 									AlwaysSetRequestIdInResponse: true,
 									HttpFilters: []*http_conn.HttpFilter{
+										{Name: "http-filter0"},
```

Comment:
Was insert-first being tested before, but just incorrectly?
T

Reply:
Yes, there were test cases before, but that's incorrect.

Reply:
https://github.com/istio/istio/blob/master/pilot/pkg/networking/core/v1alpha3/envoyfilter/listener_patch_test.go#L360